### DISCUSSION TITLE
Enable Auth Model with Auto Approve User

### DISCUSSION THREAD

**Discussion by Jacobsolawetz** [open-webui/open-webui#5025] [Ideas]
**Is your feature request related to a problem? Please describe.**

Problem - full unauthed is too much, but don't want to approve users

**Describe the solution you'd like**

Ability to set `WEBUI_AUTH` to `APPROVE_USER` or something but keep admin for first user

**Describe alternatives you've considered**

Editing source

**Comment by justinh-rahb** [comment]
Already a feature:
`DEFAULT_USER_ROLE=user`

**Reply by benlyazid** [reply]
where I can update this variable ?
